cpp/README.md

```diff
@@ -1,15 +1,23 @@
 # Alluxio C++ API
 
-### environment variables configuration
+### Introduction
+ The C++ client is implemented through JNI to native Java client. The 
```

Comment:
@Lichongjie I'm not sure about this statement. Did you run some experiment to verify it? Typically there is performance overhead when programming language barrier is crossed. For example, `libhdfs` incurs a 30% perf overhead comparing to native HDFS client.